causalpy/tests/test_integration_pymc_examples.py

```diff
@@ -258,3 +258,28 @@ def test_geolift1():
     assert isinstance(result, cp.pymc_experiments.SyntheticControl)
     assert len(result.idata.posterior.coords["chain"]) == sample_kwargs["chains"]
     assert len(result.idata.posterior.coords["draw"]) == sample_kwargs["draws"]
+
+
+@pytest.mark.integration
+def test_iv_reg():
```

Comment:
Can we also check the expected effect estimation to test the implementation itself?

Reply:
It would also be nice to parametrize this test with different priors.

Reply:
Definitely intend to add more tests of this type. Just added one to ensure I could run the testing code.

Reply:
FYI I've added a parameter recovery example to the accompanying notebook. I just wanted sure we wanted any long-running tests. It can take a little time to sample the multivariate distribution...